doc/build/dialects/index.rst

```diff
@@ -67,6 +67,8 @@ Currently maintained external dialect projects for SQLAlchemy include:
 +------------------------------------------------+---------------------------------------+
 | Amazon Redshift (via psycopg2)                 | sqlalchemy-redshift_                  |
 +------------------------------------------------+---------------------------------------+
+| Amazon Athena                                  | pyathena_                             |
```

Comment:
since this table is more or less in alphabetical order, can you move this above redshift?

Reply:
Done sir